templates/_helpers.tpl

```diff
@@ -579,3 +579,35 @@ app: "{{ template "harbor.name" . }}"
 {{- define "harbor.ingress.kubeVersion" -}}
   {{- default .Capabilities.KubeVersion.Version .Values.expose.ingress.kubeVersionOverride -}}
 {{- end -}}
+
```

Comment:
I can see the desire for a more specific and individualized configuration option, I also fear that this would that this would add more complexity to the chart and be less flexible. 

Our current solution with `CONFIG_OVERWRITE_JSON ` isn't the best design with embedded json makes it not better.  I would suggest to support a simple yaml to json converter  option,